build-docker.sh

```diff
@@ -21,7 +21,7 @@ esac
 build-silo.sh
 
 ${SUDO} docker build . -t silo:$VERSION
-${SUDO} docker tag silo:$VERSION docker.platform.cloud.coe.ic.gov/nga-r-dev/silo:$VERSION
```

Comment:
Not really any need to tag individually here... might as well just do that in the build step.